yarn.lock

```diff
@@ -975,19 +975,11 @@ append-transform@^1.0.0:
   dependencies:
     default-require-extensions "^2.0.0"
 
-aproba@^1.0.3, aproba@^1.1.1:
+aproba@^1.1.1:
```

Comment:
Please replace this file with the version that is on the Oppia repo.

Reply:
Done, thanks for reviewing.